src/components/Checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,12 @@
+import "./Checkbox.css";
+
+const Checkbox = (props) => {
+    return <div className="checkbox">
+        <div className={`checkbox__box ${props.checked === true ? "checkbox__box--checked" : ""}`}/>
```

Comment:
When you are trying to evaluate whether a variable is true, you can omit the `=== true' part, like this:

`checkbox__box ${props.checked ? "checkbox__box--checked" : ""}`